### PR TITLE
Organise styles and use theme instead of hardcoded values

### PR DESCRIPTION
**Pull request by wKich**

### CODE REVIEW

**Review by RichardW98 — APPROVED** [review]

**Review by dagda1 — APPROVED** [review]

File: plugins/parodos/src/theme.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+): ParodosPluginTheme['parodos'] => ({
+  headerBackground: '#0E2465',
+});
+
+export const createParodosTheme = <T extends BackstageTheme>(
+  theme: T & Partial<ParodosPluginTheme>,
+) => ({
+  ...theme,
+  parodos: {
+    ...createParodosThemeOverrides(theme),
```
[the remark this answers is not in the document]
Comment (wKich) [reply plugins/parodos/src/theme.ts:15]: Oh, you mean to allow specify colours from yaml file, right?

[the remark this answers is not in the document]
Comment (wKich) [reply plugins/parodos/src/theme.ts:15]: In that case you can't switch themes through backstage in UI. So I suggest to use backstage theme API https://github.com/parodos-dev/backstage-parodos/blob/6c385da3bcce340b8a6468d08299736b3eb46ed3/packages/app/src/App.tsx#L45

File: plugins/parodos/src/theme.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { BackstageTheme } from '@backstage/theme';
+import { ParodosPluginTheme } from './types';
+
+const createParodosThemeOverrides = (
+  _theme: BackstageTheme,
+): ParodosPluginTheme['parodos'] => ({
+  headerBackground: '#0E2465',
```
[the remark this answers is not in the document]
Comment (wKich) [reply plugins/parodos/src/theme.ts:7]: It's default color for parodos. And when we are overriding theme we don't know what kind of theme it is, light or dark it's just a theme.

File: plugins/parodos/src/theme.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (dagda1) [reply plugins/parodos/src/theme.ts:15]: lets just ship this and see what happens when we apply it to citibank

**Pull request merged**